### DISCUSSION TITLE
Test v0.1.0-alpha  问题反馈

### DISCUSSION THREAD

**Discussion by lencx** [lencx/Noi#2] [Ideas]
可以在这里讨论

**Comment by orienthong0304** [comment]
添加一个自定义 URL，designer.microsoft，登录之后，都是无效的，一直要重新登录，是不是自定义 URL，都没有保存登录信息的～

**Reply by lencx** [reply]
登陆无效是你的操作问题吧，具体链接是什么？

**Reply by orienthong0304** [reply]
designer.microsoft
